source/includes/aggregation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$db = $client->sample_restaurants;
+$collection = $db->restaurants;
+
+// Retrieves documents with a cuisine value of "Bakery", groups them by "borough", and
+// counts each borough's matching documents
+// start-match-group
+$pipeline = [
+    ['$match' => ['cuisine' => 'Bakery']],
+    ['$group' => ['_id' => '$borough', 'count' => ['$sum' => 1]]]
+];
+
+$cursor = $collection->aggregate($pipeline);
+
+foreach ($cursor as $doc) {
+    echo json_encode($doc) . PHP_EOL;
+}
+// end-match-group
+
+// Performs the same aggregation operation as above but asks MongoDB to explain it
+// start-explain
+$pipeline = [
+    ['$match' => ['cuisine' => 'Bakery']],
+    ['$group' => ['_id' => '$borough', 'count' => ['$sum' => 1]]]
+];
+
+$command = [
+    'explain' => [
+        'aggregate' => 'restaurants',
+        'pipeline' => $pipeline,
+        'cursor' => new stdClass()
+    ]
+];
```

Comment:
Consider the following to utilize [`MongoDB\Collection::explain()`](https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBCollection-explain/):

```php
$aggregate = new MongoDB\Operation\Aggregate(
    $collection->getDatabaseName(),
    $collection->getCollectionName(),
    $pipeline
);

$result = $collection->explain($aggregate);
```

Note that `Collection::explain()` returns a single document instead of a cursor, so there's no need to call `toArray()`.